charter.md

```diff
@@ -22,14 +22,19 @@ Server-initiated services are out of scope. The working group will first deliver
 a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
 IP proxying.
 
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP is important to ensure their continued useability.
+The working group will consider extensions to these documents that provide missing functionality or enhance performance
+for real world use cases. The initial set of extensions will be in support of UDP listening, QUIC-aware proxying, and
+datagram priorities. Extensions that help enable CONNECT-UDP or CONNECT-IP services to coordinate or work well in the
```

Comment:
I am not convinced that we have a sensible use case for datagram priorities, so I would prefer not to name it in the charter.

Reply:
I'm OK folding this into the blanket acceptance clause we have for additional extensions.

Reply:
+1, I'm not hearing anyone saying they want to work on datagram priorities

Reply:
I think the use case exists and I would work on it. But I can live with it not being mentioned in the charter explicitly, and it running through the standard process of adoption as an extension.

Reply:
Removed, but also comfortable with doing this as clearing the bar laid out in the non-explicitly mentioned set